chain-spec-generator/src/relay_chain_specs.rs

```diff
@@ -348,7 +43,9 @@ pub fn polkadot_development_config() -> Result<Box<dyn ChainSpec>, String> {
 		.with_name("Polakdot Development")
 		.with_id("polkadot-dev")
 		.with_chain_type(ChainType::Development)
-		.with_genesis_config_patch(polkadot_development_config_genesis())
+		.with_genesis_config_patch(
```

Comment:
You could use: [`with_genesis_config_preset_name`](https://github.com/paritytech/polkadot-sdk/blob/6db203822418eee666f44ccaed3cd96fdfa27351/substrate/client/chain-spec/src/chain_spec.rs#L404).

This would allow to fetch the preset from the wasm blob, w/o necessity to natively compile the runtime,